powerpacks.go

```diff
@@ -390,6 +476,10 @@ func (service *PowerpackService) Buy_add_number(phoneParam BuyPhoneNumberParam)
 	payload := RentNumber{
 		Rent: "true",
 	}
+
+	if phoneParam.Service != "" {
```

Comment:
This condition is unnecessary as the default if not set is empty string. You can directly initialize this:

```go
payload := RentNumber{
    ...
    Service: phoneParam.Service,
}
```

Reply:
fixed